mapie/tests/test_metrics.py

```diff
@@ -597,3 +610,59 @@ def test_classification_coverage_score_v2_ypredset_invalid_shape() -> None:
         classification_coverage_score_v2(
             np.expand_dims(y_true_class, axis=1), y_pred_set[:, 0]
         )
+
+
+def test_mu_valid() -> None:
+    # Test mu valid
+    assert np.allclose(round(cwc(y_preds[:, 0], y_preds[:, 1], y_preds[:, 2],
+                                 eta=30, mu=0.9), 2), 0.48)
+
+
+def test_mu_invalid() -> None:
+    # Test mu invalid
+    try:
+        assert cwc(y_preds[:, 0], y_preds[:, 1], y_preds[:, 2], eta=30, mu=-1)
+    except ValueError:
+        assert True
+
+
+def test_high_eta() -> None:
+    # Test high eta
+    np.allclose(round(cwc(y_preds[:, 0], y_preds[:, 1], y_preds[:, 2], eta=30,
+                          mu=0.9), 2), 0.48)
+
+
+def test_low_eta() -> None:
+    # Test low eta
+    np.allclose(round(cwc(y_preds[:, 0], y_preds[:, 1], y_preds[:, 2],
+                          eta=0.01, mu=0.9), 2), 0.65)
+
+
+def test_negative_eta() -> None:
+    # Test negative eta
+    np.allclose(round(cwc(y_preds[:, 0], y_preds[:, 1], y_preds[:, 2], eta=-1,
+                          mu=0.9), 2), 0.65)
+
+
+def test_null_eta() -> None:
+    # Test null eta
+    np.allclose(round(cwc(y_preds[:, 0], y_preds[:, 1], y_preds[:, 2], eta=0,
+                          mu=0.9), 2), 0.65)
```

Comment:
SUGGESTION: merge all tests in one
```suggestion
def test_valid_eta() -> None:
    """Test different values of eta in cwc metric."""
    y, y_low, y_up = y_preds[:, 0], y_preds[:, 1], y_preds[:, 2]
    np.testing.assert_allclose(
        cwc(y, y_low, y_up, eta=30, mu=0.9), 0.48, rtol=1e-2
    )
    np.testing.assert_allclose(
        cwc(y, y_low, y_up, eta=0.01, mu=0.9), 0.65, rtol=1e-2
    )
    np.testing.assert_allclose(
        cwc(y, y_low, y_up, eta=-1, mu=0.9), 0.65, rtol=1e-2
    )
    np.testing.assert_allclose(
        cwc(y, y_low, y_up, eta=0, mu=0.9), 0.65, rtol=1e-2
    )
```